### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by Oleg-Chashko**
Small fixes

### CODE REVIEW

File: Translations/Web/lang_ru.php [hunk 1]
```diff
@@ -4,7 +4,7 @@
 //
 // Language: Russian
 $content_language_locale = "ru";
-// Translated by NickKaramoff (nick@karamoff.ru)
+// Translated by Nick Karamoff (nick@karamoff.ru)
```
Comment (aonez) [inline Translations/Web/lang_ru.php:7]: This one is from the author so better leave it as he likes :)

https://github.com/aonez/Keka/pull/312/files#diff-f2232528f4d142de92ba0393e18d6ccfR6

File: Translations/Web/lang_bg.php [hunk 2]
```diff
@@ -22,7 +22,7 @@
 $content_title = "macOS файлов архиватор";
 
 // Download
-$content_plataform = "Изисква $version_replace или по-нова";
```
Comment (aonez) [inline Translations/Web/lang_bg.php:25]: I'm ashamed 🤦🏻‍♂️

File: README.md [hunk 3]
```diff
@@ -34,7 +34,7 @@ Also head to [Releases](https://github.com/aonez/Keka/releases) if you need a co
 Sadly, the source code of Keka 1.0 and newer will be not public due some issues in the past (this is [one of the reasons](http://web.archive.org/web/20180615184129/http://trac.keka.io/ticket/14)).
 
 ### Legacy info
-Keka started in [SourceForge](http://sourceforge.net/projects/keka). It will be rarely updated and issues in the [Tickets](https://sourceforge.net/p/keka/_list/tickets) may be replied, but [Issues](https://github.com/aonez/Keka/issues) here is the main cannal.
```
Comment (aonez) [inline README.md:37]: 🥶

File: Translations/Web/lang_bg.php (hunk 2 above)
Comment (aonez) [reply Translations/Web/lang_bg.php:25]: From the Spanish "plataforma"...

**Review by aonez — CHANGES_REQUESTED** [review]

File: Translations/Web/lang_ru.php (hunk 1 above)
Comment (Oleg-Chashko) [reply Translations/Web/lang_ru.php:7]: > This one is from the author so better leave it as he likes :)

Here is an example of the fact that the author himself is not yet sure how to write his first and last name together or separately :)
**Translators_html**
Translators_html

File: Translations/Web/lang_bg.php (hunk 2 above)
Comment (Oleg-Chashko) [reply Translations/Web/lang_bg.php:25]: > From the Spanish "plataforma"...

I immediately realized that you took it from your native language :)

**Comment by Oleg-Chashko** [comment]
Sorry to be off topic. I haven't worked with Git. For example, you do not accept some changes, what should I do with my changed files on my Mac?

**Comment by aonez** [comment]
> Sorry to be off topic. I haven't worked with Git. For example, you do not accept some changes, what should I do with my changed files on my Mac?

You can just revert to the original source, either manually or with an option in your Git app. Then commit and push the change.

**Comment by aonez** [comment]
@Oleg-Chashko are you going to reopen this? Note that if you have an active pull request and make changes to your repo it will automatically update, so there's no need to close and create a new pull request.

**Comment by Oleg-Chashko** [comment]
@aonez
@Oleg-Chashko are you going to reopen this? Note that if you have an active pull request and make changes to your repo it will automatically update, so there's no need to close and create a new pull request.

Sorry, I have to learn to work with Git. And I want to learn how to work correctly with branches and etc. If you can, remove the merge request. I am currently building everything back, with branches for each fix. Once again, I'm sorry and understand me, I'm learning.

**Comment by aonez** [comment]
No problem at all @Oleg-Chashko! I don't think you need branches for this, though, but do it if you want to try/learn.

Salut!

**Pull request closed without merging**